src/main/java/frc/robot/commands/vision/PlayerStationAutoAlign.java

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.vision;
+import frc.robot.Constants;
+import frc.robot.subsystems.DrivetrainSubsystem;
+import frc.robot.subsystems.Vision;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class PlayerStationAutoAlign extends CommandBase {
+  /** Creates a new PlayerStationAutoAlign. */
+  private final Vision vision;
+  private final DrivetrainSubsystem drivetrainSubsystem;
+  public PlayerStationAutoAlign(DrivetrainSubsystem drivetrainSubsystem, Vision vision) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    this.vision = vision;
+    this.drivetrainSubsystem = drivetrainSubsystem;
+    addRequirements(this.vision, this.drivetrainSubsystem);
```

Comment:
Vision is a special subsystem for us that can be used by multiple commands therefore we don’t need to add it to the requirements

Reply:
I removed it from the addRequirements call, is this good practice?